imagelab/DynaPanel.java

```diff
@@ -20,96 +20,105 @@
     public static DynaPanel getDynPan() {
-        if(dynPanSingleton == null) {
-            dynPanSingleton = new DynaPanel((Image)null);
+        if (dynPanSingleton == null) {
+            dynPanSingleton = new DynaPanel((Image) null);
         }
         dynPanSingleton.setVisible(true);
         return dynPanSingleton;
     }
 
-    // Instance Variables
+    /**Instance Variable for background color. */
     private Color backgroundColour;
+    /**Instance Variable for the panel image. */
     private Image panelImage;
+    /**Instance Variable for the graphic. */
     private Graphics graphic;
-    
+
     /**
      * Create a default DynaPanel.
      */
     public DynaPanel() {
         super(true);
         panelImage = null;
         System.out.println("DynaPanel: default panelImage = " + panelImage);
-   }
+    }
 
     /**
      * Create a DynaPanel.
+     *
      * @param img image to be displayed
      */
-    public DynaPanel(Image img) {
+    public DynaPanel(final Image img) {
         super(true);
         panelImage = img;
         System.out.println("DynaPanel: panelImage = " + panelImage);
     }
 
     /**
      * Create a DynaPanel.
+     *
      * @param imp image to be displayed
      */
-    public DynaPanel(ImgProvider imp) {
+    public DynaPanel(final ImgProvider imp) {
         super(true);
-        int width  = imp.getWidth();
+        int width = imp.getWidth();
         int height = imp.getHeight();
         panelImage = imp.getImage();
         System.out.println("DynaPanel: panelImage = " + panelImage);
     }
 
     /**
      * Draw a given image onto the canvas.
+     *
      * @param image the image object to be drawn on the canvas
      */
-    public void newImage(Image image) {
+    public void newImage(final Image image) {
         System.out.println("DynaPanel:draw(i): image = " + image);
         panelImage = image;
     }
 
     /**
      * Draw a given image onto the panel and change the frame title.
-     * @param image the image object to be drawn on the canvas
-     * @param title the text to use as the frame title
+     *
+     * @param imp the image object to be drawn on the canvas
      */
-    public void draw(ImgProvider imp) {
+    public void draw(final ImgProvider imp) {
         newImage(imp.getImage());
     }
-  
+
 
     /**
      * Wait for a specified number of milliseconds before finishing.
      * This provides an easy way to specify a small delay which can be
      * used when producing animations.
-     * @param  milliseconds  the number 
+     *
+     * @param milliseconds the number
      */
-    public void wait(int milliseconds) {
+    public void wait(final int milliseconds) {
         try {
             Thread.sleep(milliseconds);
-        } 
-        catch (Exception e) {
+        } catch (Exception e) {
             // ignoring exception at the moment
         }
     }
 
     /**
      * Dynamic (re-)paint.
+     *
+     * @param g Graphic being used to draw image
```

Comment:
Confused use of term "Graphic".  See API for java.awt.Graphics to determine more appropriate description of the parameter.